app/src/main/java/com/github/mik629/android/fundamentals/domain/repositories/MoviesRepository.kt

```diff
@@ -1,7 +1,9 @@
 package com.github.mik629.android.fundamentals.domain.repositories
 
-import com.github.mik629.android.fundamentals.domain.model.MovieItem
+import com.github.mik629.android.fundamentals.domain.model.Movie
 
 interface MoviesRepository {
-    suspend fun getMovies(): List<MovieItem>
+    suspend fun getMovies(): List<Movie>
```

Comment:
separete Movie and MovieDetails

Reply:
Hmm, they were separated and on one of first reviews you'd said that they're too much alike, let's merge them 😁